src/main/java/jsr223/docker/compose/bindings/MapBindingsAdder.java

```diff
@@ -34,23 +34,21 @@
  */
 package jsr223.docker.compose.bindings;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this package name to match the regular expression '^[a-z]+(\.[a-z][a-z0-9]*)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS00120)